controllers/v1/userExtensionController.js

```diff
@@ -0,0 +1,247 @@
+          message: error.message || "Oops! something went wrong.",
+          errorObject: error
+        })
+      }
+
+    })
+  }
+
+
+    /**
+  * @api {post} /user-management-service/api/v1/userExtension/create add User
+  * @apiVersion 1.0.0
+  * @apiName Create User 
+  * @apiGroup User Extension
+  * @apiParam {reqeuestBody} consist of body of the request
+  * @apiSampleRequest /user-management-service/api/v1/userExtension/create
+  * {
+  *   "firstName":"test",
+  *   "lastName":"test",
+  *   "email":"testUser12333@gmail.com",
+  *   "phoneNumber":"1234567890",
+  *   "userName":"testUser33@1234",
+  *   "state":{
+  *      "label":"Karnataka",
+  *      "value":"5d6609ef81a57a6173a79e7a"
+  *   },
+  *   "organisation":{
+  *      "label":"ShikshaLokamDev",
+  *      "value":"0125747659358699520"
+  *    },
+	*  "roles":[
+  *   {
+  *      "label":"ORG_ADMIN",
+  *      "value":"ORG_ADMIN"
+  *   }
+  *  ],
+  * "dateofbirth":"1994-0-01"
+  * }
+  * @apiUse successBody
+  * @apiUse errorBody
+  * @apiParamExample {json} Response:
+  * 
+  * {
+  *   "message": "User created successfully",
+  *   "status": 200,
+  *    "result": [ 
+  *     {  
+  *         "response": "SUCCESS",
+  *         "userId": ""
+  *     }
+  *  ]
+  * }
+  */
+
+  create(req) {
+    return new Promise(async (resolve, reject) => {
+
+      try {
+        let userCreate = await userExtensionHelper.create(req.body,req.userDetails.userToken,req.userDetails.userId);
+        return resolve(userCreate);
+
+      } catch (error) {
+        return reject({
+          status: error.status || httpStatusCode.internal_server_error.status,
+          message: error.message || "Oops! something went wrong.",
+          errorObject: error
+        })
+      }
+    })
+  }
+
+
+   /**
+   * @api {post} /user-management-service/api/v1/userExtension/activateUser
+   * @apiVersion 1.0.0
+   * @apiName To activate user
+   * @apiGroup User Extension
+   * @apiParam {reqeuestBody} consist of body of the request
+   * @apiSampleRequest /user-management-service/api/v1/userExtension/activateUser
+   * {
+   *   userId:""
+   * }
+   * @apiUse successBody
+   * @apiUse errorBody
+   * @apiSampleResponse
+   * {
+   *    "message": "User activated successfully",
+   *    "status": 200,
+   *    "result": {
+   *      "response": "SUCCESS"
+   *    }
+   * }
+   */
+  activateUser(req) {
+    return new Promise(async (resolve, reject) => {
+
+      try {
+
+        let userBlockInfo = await userExtensionHelper.activateUser(req.body.userId, req.userDetails.userToken);
+        return resolve(userBlockInfo);
+
+      } catch (error) {
+        return reject({
+          status: error.status || httpStatusCode.internal_server_error.status,
+          message: error.message || "Oops! something went wrong.",
+          errorObject: error
+        })
+      }
+    })
+  }
+
+  /**
+   * @api {post} /user-management-service/api/v1/userExtension/inActivateUser
```

Comment:
@deepakr123 , @rakeshSgr - inActivateUser can be just inactivate. User is already there in the main URL

Reply:
resolved